circleci/Dockerfile

```diff
@@ -88,5 +88,9 @@ COPY requirements.txt /
 COPY requirements /requirements
 RUN python3 -m pip install -r requirements.txt -r requirements/circleci.txt
 
+# External calls configuration
+COPY .awsconfig .aws/config
+COPY .curlrc .wgetrc ./
```

Comment:
Shouldn't the destination be more explicit? In case we change the `WORKDIR` beforehand for whatever reason